connector/countconnector/ottl.go

```diff
@@ -0,0 +1,111 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package countconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/countconnector"
```

Comment:
I think you need to check the rebase again, the countconnector uses `filterottl` now and doesn't need the `noop` function anymore.

Reply:
Weird, I thought I deleted it while rebasing. It's gone now, thanks for catching that.